src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -62,7 +63,7 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         model.addStudent(toAdd);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, Messages.format(toAdd)));
+        return new CommandResult(String.format(MESSAGE_SUCCESS, Messages.format(toAdd)), CommandType.ADDSTUDENT);
```

Comment:
It may be easier to reuse COMMAND_TYPE here rather than explicit usage of CommandType.ADDSTUDENT. Same for the other command classes.

Reply:
Good Catch. Changed Accordingly.